src/index.html

```diff
@@ -1,12 +1,404 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo</title>
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+        class="header__logo-image"
+        src="images/logo.png"
+        alt="header logo"
+        >
+      </a>
+      <nav class="header__group">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#apple" class="header__link is-active">Apple</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#samsung" class="header__link">Samsung</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#smartphones" class="header__link">Smartphones</a>
+          </li>
+
+          <li class="header__item">
+            <a
+            href="#laptops-&-computers"
+            class="header__link"
+            data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#gadgets" class="header__link">Gadgets</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#tablets" class="header__link">Tablets</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#photo" class="header__link">Photo</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#video" class="header__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="container">
+        <div class="card" data-qa="card">
+          <div class="card__image"></div>
+
+          <p class="card__name">
```

Comment:
`h2` since it's a card title
```suggestion
          <h2 class="card__name">
```